modules/stanford_events/stanford_events.module

```diff
@@ -275,3 +276,10 @@ function stanford_events_preprocess_pattern_events_list(&$variables) {
     $variables['end_month_full'] = date('F', strtotime($end_month));
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function stanford_events_form_node_stanford_event_form_alter(array &$form, FormStateInterface $form_state) {
+  $form['su_event_sponsor']['widget']['add_more']['#value'] = t("Add another sponsor");
```

Comment:
I'm going to push back on this. This opens a whole bunch of other similar button changes and increased maintenance. Maybe take a look at [change_labels](https://www.drupal.org/project/change_labels) module.
> Allows to change the "Add another item" per widget in multivalued fields.

Reply:
In fact, looking at the source code of that module, it's really pretty clean and I now know a few more event hooks we can use.